views/layouts/authentication.html

```diff
@@ -9,9 +9,10 @@
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
+    {{ assets_css "/static/css/application.css" }}
   </head>
 
-  <body>
+  <body class="{{.ControllerName}} {{.ActionName}} container">
```

Comment:
Avoid using `container` on the whole body. Prefer adding it to each main element on the page and prefer to use the SCSS mixin `@include make-container()`